homeassistant/components/kodi/media_player.py

```diff
@@ -628,6 +629,23 @@ def media_album_artist(self):
 
         return None
 
+    @property
+    def extra_state_attributes(self) -> dict[str, str | None]:
+        """Return the state attributes."""
+        state_attr: dict[str, str | None] = {}
+        if self.state == MediaPlayerState.OFF:
+            return state_attr
+
+        hdr_type = (
+            self._item.get("streamdetails", {}).get("video", [{}])[0].get("hdrtype")
+        )
+        if hdr_type == "":
+            state_attr["hdr_type"] = "sdr"
+        else:
+            state_attr["hdr_type"] = hdr_type
```

Comment:
```suggestion
            state_attr["dynamic_range"] = hdr_type
```

Reply:
Can we also add translations for this state attribute? example:
https://github.com/home-assistant/core/blob/f94167a4bbf3e5620fd5ba1e31e6794df90052dd/homeassistant/components/youtube/strings.json#L44-L47

Reply:
@joostlek how to do this if `_attr_name = None` and there is no translation key for the media_player entity (since it is the only single entity of the integration)?

Reply:
just come up with a translation key, "media_player" would for example work

Reply:
But the translation key takes presidents over the _attr_name. Or can I just not include the name in strings.json?

Reply:
> the translation key takes presidents over the _attr_name

No, it doesn't

Reply:
I added the translation and tested it.